WordPress/src/main/java/org/wordpress/android/push/NotificationsProcessingService.java

```diff
@@ -0,0 +1,402 @@
+package org.wordpress.android.push;
```

Comment:
Would you mind to run Code Inspector on this file, and fix some of the suggestions?